### PR TITLE
LUN-96: Use Json for german labels

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

File: src/constants/labels.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      "description": "Wähle das richtige Wort"
+    },
+    "articleSingleChoice": {
+      "title": "Artikel üben",
+      "description": "Wähle den richtigen Artikel"
+    },
+    "write": {
+      "title": "Wort schreiben",
+      "description": "Wort mit Artikel eingeben",
+      "placeholder": {
```
Comment (klinzo) [inline src/constants/labels.json:31]: I'd not use these technical separations like `placeholder` and `button`. If you use another UI element for the same purpose it does not make sense anymore, so I think these constants should be completely independent of the concrete implementation. I guess I'd just omit the additional separation and put it one level up (directly in `write` in this example).

File: src/constants/data.ts [hunk 2]
```diff
@@ -28,29 +29,29 @@ export interface ExerciseType {
 export const EXERCISES: ExerciseType[] = [
   {
     key: ExerciseKeys.vocabularyOverview,
-    title: 'Vokabelübersicht',
-    description: 'Alle Wörte anschauen',
+    title: labels.exercises.overview.title,
+    description: labels.exercises.overview.description,
```
Comment (klinzo) [inline src/constants/data.ts:33]: I think we should be consistent with naming of the exercises in all constants, labels and files... I am really confused which exercise is which :see_no_evil:

File: src/constants/labels.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
+{
+  "overview": {
+    "welcome": "Willkommen bei Lunes!\nLerne Vokabeln für deinen Beruf.",
+    "unit": "Lektion",
+    "units": "Lektionen",
+    "word": "Wort",
+    "words": "Wörter",
+    "exercises": "Übungen"
+  },
+  "header": {
```
Comment (klinzo) [inline src/constants/labels.json:10]: I'd just move those to overview/home/... from above as well, see comment below.

File: src/constants/labels.json [hunk 4]
```diff
@@ -0,0 +1,73 @@
+{
+  "overview": {
```
Comment (klinzo) [inline src/constants/labels.json:2]: I think `main`, `home` or `general` would be more fitting. It also kind of conflicts with the exercise `overview`.

File: src/constants/labels.json [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  },
+  "header": {
+    "overview": "Übersicht",
+    "overviewExercises": "Übersicht der Übungen",
+    "cancelExercise": "Übung beenden"
+  },
+  "exercises": {
+    "overview": {
+      "title": "Vokabelübersicht",
+      "description": "Alle Wörte anschauen"
```
[the remark this answers is not in the document]
Comment (SteffiStuffel) [reply src/constants/labels.json:18]: done

File: src/constants/data.ts (hunk 2 above)
Comment (SteffiStuffel) [reply src/constants/data.ts:33]: yes, there is a ticket for this.

File: src/constants/labels.json (hunk 1 above)
Comment (SteffiStuffel) [reply src/constants/labels.json:31]: done.

File: src/constants/labels.json (hunk 4 above)
Comment (SteffiStuffel) [reply src/constants/labels.json]: done

File: src/constants/labels.json (hunk 3 above)
Comment (SteffiStuffel) [reply src/constants/labels.json]: done

**Review by klinzo — APPROVED** [review]

**Pull request merged**